domains/auth/Auth.API/API/Controllers/InvalidateController.cs

```diff
@@ -14,12 +12,12 @@ public class InvalidateController : ControllerBase
 {
     private readonly IOidcService oidcService;
     private readonly IValidator<AuthState?> validator;
-    private readonly ICryptography cryptography;
+    private readonly ICryptographyFactory cryptography;
```

Comment:
Couldnt this be of type `ICryptography` and then set it to `StateCryptography` in the constructor